src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -80,64 +80,64 @@
 
   </Target>
 
-  <Target Name="_RazorResolveSourceFiles">
+  <Target Name="_ResolveRazorCoreGenerateInputs">
+
     <ItemGroup>
-      <RazorCompile Include="@(Content)" Condition="'%(Content.Extension)'=='.cshtml'" />
-      <_RazorGenerated Include="@(RazorCompile->'$(RazorGenerateOutputPath)%(RelativeDir)%(Filename).cs')">
-        <Source>%(Identity)</Source>
-      </_RazorGenerated>
+      <RazorGenerate Include="@(Content)" Condition="'%(Content.Extension)'=='.cshtml'" />
```

Comment:
RazorGenerate = .cshtml inputs to code generator
RazorCompile = .cs inputs to CSC

This was you can add your own files if you want to include them, like assembly attributes and whatnot.